gamestate.js

```diff
@@ -9,6 +9,45 @@ modeRef.on('value', function (v) {
         $("#" + mode).show();
     }
 });
+
+// Gatherer card images are low-res and blocky and ugly, so if we can, we want to use magiccards.info images.
+// We need to translate from what Gatherer calls the set to what magiccards.info calls the set, though.
+var gathererToMagiccardsInfo = [];
+$.get("https://api.magicthegathering.io/v1/sets",
+    function(data) {
+        gathererToMagiccardsInfo = _.object(
+            _.map(data.sets, function(x) { return x.gathererCode || x.code; }),
+            _.map(data.sets, function(x) { return x.magicCardsInfoCode; }));
+    });
+
+function fillFeaturedCard(cardName, featuredCardSelector, handSelector) {
+    if (cardName != "") {
+        $(handSelector).hide();
```

Comment:
I'd put the hide() and show() calls in the request callback to mask the latency.